### PR TITLE
Add risk translation

### PR DESCRIPTION
**Pull request by robekeane**
#25 @xiaoya-Esther @eyehwan

### CODE REVIEW

**Comment by eyehwan** [comment]
Looks good to me!

File: chinese/wg-risk/focus-areas/business-risk/bus-factor.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# **巴士系数**
+
+问题：如果最活跃的人离开，项目会产生多大风险？
+
+## 描述  
+巴士系数是一个引人注目的指标，因为它形象化了“我们失去多少贡献者就会使项目停滞？”的问题。假设这些贡献者被一辆公共汽车撞了，（换个令人容易接受点的说法是，有多少人必须在彩票中获胜并决定继续前进）。
```
Comment (xiaoya-Esther) [inline chinese/wg-risk/focus-areas/business-risk/bus-factor.md:6]: “有多少人中了彩票并决定离开项目” 会不会更好理解？“move on”是一个比较西方的表达，我的理解是不再继续待在这个项目了

File: chinese/wg-risk/focus-areas/business-risk/bus-factor.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+巴士系数是贡献超过整个项目50%的最少人数。
+
+
+## 目标  
+* 确定项目工作在贡献者之间分布的范围。
+* 确定项目中负责大部分工作的关键人员。
+
+## 实现  
+巴士系数的公式是一个百分比计算-50%将是我们的阈值-
```
Comment (xiaoya-Esther) [inline chinese/wg-risk/focus-areas/business-risk/bus-factor.md:16]: “ 巴士系数的公式是一个百分比计算 - 这里我们使用 50% 作为阈值 - 然后将每个贡献者的贡献按降序排序，直到贡献总和到达门槛”

File: chinese/wg-risk/focus-areas/licensing/license-coverage.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
 # 许可证覆盖
 
-问题：代码库有多少已声明许可证？
+问题：代码仓库有多少已声明许可证？
```
Comment (xiaoya-Esther) [inline chinese/wg-risk/focus-areas/licensing/license-coverage.md:3]: According to the [english-chinese-comparison-table](https://github.com/chaoss/translations/blob/main/chinese/english-chinese-comparison-table.md), "repository" is currently translated to "代码仓", do we have better ideas?

Comment (eyehwan) [reply chinese/wg-risk/focus-areas/licensing/license-coverage.md:3]: Prefer "代码仓" :)

File: chinese/wg-risk/focus-areas/business-risk/bus-factor.md (hunk 1 above)
Comment (robekeane) [reply chinese/wg-risk/focus-areas/business-risk/bus-factor.md:6]: @xiaoya-Esther 嗯，小雅老师的表达更准确

File: chinese/wg-risk/focus-areas/licensing/license-coverage.md (hunk 3 above)
Comment (robekeane) [reply chinese/wg-risk/focus-areas/licensing/license-coverage.md:3]: @xiaoya-Esther @eyehwan ok

File: chinese/wg-risk/focus-areas/business-risk/bus-factor.md (hunk 2 above)
Comment (robekeane) [reply chinese/wg-risk/focus-areas/business-risk/bus-factor.md:16]: @xiaoya-Esther 简洁！

**Comment by robekeane** [comment]
@xiaoya-Esther @eyehwan I update the PR with your comments

**Review by xiaoya-Esther — APPROVED** [review]
Thanks. LGTM

**Pull request merged**